templates/generate_tiled_regions.ipynb

```diff
@@ -0,0 +1,167 @@
+{
```

Comment:
<p>The moly path will always be the same, it's not run specific. Therefore, I think for this notebook, we don't need the set prefix. Instead, the output file name should default to be the input file name with <code>_tiled_fovs</code>  appended to the end</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/angelolab/toffy/pull/41/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='5'/>

Reply:
<p>Ok sounds good. I'll make the following changes:</p><p> </p><ul><li><code>region_corners_path</code>:  default to <code>region_corners.json</code>  without the prefix</li><li><code>tiled_region_fovs_path</code>: default to <code>region_corners_path + '_tiled_fovs.json</code> </li><li><code>moly_path</code>: default to <code>moly_point.json</code> <span class="ql-cursor">﻿</span></li></ul>